daras_ai_v2/language_model.py

```diff
@@ -819,6 +874,13 @@ def run_language_model(
     ), "Pleave provide exactly one of { prompt, messages }"
 
     model: LargeLanguageModels = LargeLanguageModels[str(model)]
+
+    if model.is_deprecated:
+        if model.redirect_to:
+            return run_language_model(**locals() | {"model": model.redirect_to})
+        else:
+            raise UserError(f"Model {model} is deprecated.")
```

Comment:
this is new behavior for deprecated models without a `redirect_to` option. raising `UserError` here.

earlier, the behavior was to attempt to run the model regardless.